### ISSUE TITLE
rinoh shouldn't require absolute paths for the template and style sheet

### ISSUE THREAD

**Issue by kfrajtak** [brechtm/rinohtype#253]
When I run `rinoh -s mystyle.rst index.rst` I get 

```
ValueError: StyleSheetFile path should be absolute: mystyle.rst
```

`mystyle.rst` is in current directory

Please add CLI example to http://www.mos6581.org/rinohtype/master/basicstyling.html chapter.

**Comment by brechtm** [comment]
This is indeed not the intended behavior. This is a regression introduced with recent changes.

**Comment by brechtm** [comment]
Fixed in #266
